sidekick_core/lib/src/commands/update_command.dart

```diff
@@ -78,10 +78,20 @@ class UpdateCommand extends Command {
       pubspecKeys: ['dependencies', 'sidekick_core'],
       newMinimumVersion: to,
     );
-    _dartCommand(
-      ['pub', 'get'],
-      workingDirectory: Repository.requiredCliPackage,
-    );
+    try {
+      _dartCommand(
+        ['pub', 'get'],
+        workingDirectory: Repository.requiredCliPackage,
+        progress: Progress.devNull(),
```

Comment:
Add this to the first `pub get`, too! 👏

Reply:
I think on the first `pub get` `Progress.printStdErr()` would be better because that `pub get` needs to succeed and we should print the error output e.g. if it fails because there is no internet connection